### ISSUE TITLE
Developer Tools console is very noisy

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#1452]
Found in https://testpilot.firefox.com/__version__ (8ad00242ebb8c075e50ec39faf1cc9efc28a7e2f):

``` json
{ "source": "https://github.com/mozilla/testpilot",
  "commit": "8ad00242ebb8c075e50ec39faf1cc9efc28a7e2f",
  "build": "https://circleci.com/gh/mozilla/testpilot/1143",
  "version": "2016-09-26-1" }
```

---
### Steps to reproduce:
1. Go to https://testpilot.firefox.com/experiments/page-shot (currently 404s until new experiments released).
2. Open Developer Tools console.
### Actual results:

![firefox_test_pilot](https://cloud.githubusercontent.com/assets/557895/18884088/d8d842f0-849a-11e6-9e98-9e9b71936742.png)

> "Warning: It looks like you're using a minified copy of the development build of React. When deploying React apps to production, make sure to use the production build which skips development warnings and is faster. See https://fb.me/react-minification for more details."
> 
> "You are currently using minified code outside of NODE_ENV === 'production'. This means that you are running a slower development build of Redux. You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify or DefinePlugin for webpack (http://stackoverflow.com/questions/30030031) to ensure you have the correct code for your production build."
> 
> "TypeError: d is undefined [Learn More]"

Plus oodles of console logs, which are hard to debug since the code is minimized.

**Comment by lmorchard** [comment]
I think this is a dupe of / will be addressed by #1349

**Comment by ghost** [comment]
Closing as dupe of #1349. thanks
